components/button/button.html

```diff
@@ -1,110 +1,181 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Paper UI | Button</title>
-    <link rel="stylesheet" href="button.css">
-    <link rel="shortcut icon" href="../../images/logo.svg" type="image/x-icon">
+    <link rel="stylesheet" href="button.css" />
+    <link
+      rel="shortcut icon"
+      href="../../images/logo.svg"
+      type="image/x-icon"
+    />
     <link
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
-</head>
-<body>
+  </head>
+  <body>
     <nav class="nav">
```

Comment:
you can use git diff once before pushing and try to revert such small changes to no changes. it will help in keeping a track of relevant changes and also help to avoid redundant merge conflicts.

Reply:
meanwhile it is good for now can keep a track of it from next time

Reply:
Thanks!!